test/Spec.hs

```diff
@@ -11,5 +11,6 @@ main = do
     print typeSpec_Normalize
     print typeSpec_DivRat
     print typeSpec_UnitCalculation
+    print typeSpec_ShowUnits
```

Comment:
I thought a little bit more about this. I think it's better to refactor `TypeSpec` in a way you did for `tasty` tests. Just export single function `runTypeSpecTests`, which just does all this printing. Wouldn't you mind to refactor this?